### ISSUE TITLE
Missing Focus Ring on Pattern Inserter from Sidebar

### ISSUE THREAD

**Issue by jeryj** [WordPress/gutenberg#60211]
Some patterns do not have a focus style. In this image, focus was moved to the "Standard" pattern (the tooltip is visible), but the focus ring is not present. All patterns should have some sort of focus indicator.

<img width="629" alt="" src="https://github.com/WordPress/gutenberg/assets/967608/81b9b39e-fce3-4bbc-b651-b4988c41f779">

**Comment by getdave** [comment]
This is "In Progress" but there is no linked PR.

**Comment by jeryj** [comment]
@getdave Thanks for the catch. There's a PR ready for review at https://github.com/WordPress/gutenberg/pull/60881

**Comment by getdave** [comment]
No problem. I just wondered if there a PR I could review 😄
